include/crm/common/output_internal.h

```diff
@@ -27,7 +27,7 @@ extern "C" {
 #  include <glib.h>
 #  include <crm/common/results.h>
 
-#  define PCMK__API_VERSION "2.3"
+#  define PCMK__API_VERSION "2.9"
```

Comment:
Did you mean to go with .9 instead of .4 here?

Reply:
Yes, it's the latest schema version (xml/api/api-result-2.9.rng)